typescript/libs/popup_manager.py

```diff
@@ -262,14 +262,28 @@ def encode(str, kind):
                     if param["documentation"] else "")
         else:
             activeParam = ''
+           
+        theme_styles = self.get_theme_styles()
 
         return {"signature": signature,
                 "description": description,
                 "activeParam": activeParam,
                 "index": "{0}/{1}".format(self.signature_index + 1,
                                           len(self.signature_help["items"])),
                 "link": "link",
-                "fontSize": PopupManager.font_size}
+                "fontSize": PopupManager.font_size,
+                "typeColor": theme_styles["type"]["foreground"],
+                "nameColor": theme_styles["name"]["foreground"],
+                "paramColor": theme_styles["param"]["foreground"],
+                "textColor": theme_styles["text"]["foreground"]}
+
+    def get_theme_styles(self):
+        return {
+            "type": self.current_view.style_for_scope("entity.name.type.class.ts"),
```

Comment:
Kind of unfortunate [we don't do anything for keywords](https://github.com/Microsoft/TypeScript-Sublime-Plugin/blob/3f58e5a7a9364e3ad551e7ed4a47a0fd0cb9b608/typescript/libs/popup_manager.py#L206) but I guess it's fine for now. Feel free to send a follow-up.

Reply:
I've separated `keyword` chunks out into their own type and added keyword styling to them. I'm not sure what code I can test it on, though, I don't actually see keyword scopes anywhere in signature popups so far just testing out a few different cases.